glue_jupyter/bqplot/common/tools.py

```diff
@@ -207,12 +219,6 @@ def on_selection_change(self, *args):
             if self.finalize_callback is not None:
                 self.finalize_callback()
 
-    def activate(self):
-        with self.viewer._output_widget:
-            self.interact.selected_x = None
-            self.interact.selected_y = None
-        super().activate()
```

Comment:
I don't think it makes sense to move these to the base class because ``BqplotXRangeMode`` for example should also inherit from the base class and only has selection in one direction. Having said that I wonder if this code (setting the selected variables to None) should be in the deactivate methods instead of activate.

Reply:
I have no idea what this thing is doing, so just tell me what to move where and I will do it.

Reply:
But I don't understand your comment here. I didn't move it to the global base class, only to `BqplotSelectionTool`, which only rectangle and circle inherit from. How will this affect `BqplotXRangeMode` in any way?

Reply:
My point was more that ``BqplotXRangeMode`` is also a selection tool so should also inherit from the base class - however the code I highlighted above is for 2-d selections and won't work for 1-d selectors such as ``BqplotXRangeMode``